mission_control/src/behaviors/fixed_rudder.cpp

```diff
@@ -79,28 +79,55 @@ BT::NodeStatus MoveWithFixedRudder::behaviorRunningProcess()
 {
   if (!goalHasBeenPublished_)
   {
-    publishGoalMsg();
-    goalHasBeenPublished_ = true;
+    auto res = getInput<std::string>("behavior_time");
+    if (!res)
+    {
+      ROS_ERROR_STREAM("error reading port [behavior_time]:" << res.error());
+      setStatus(BT::NodeStatus::FAILURE);
+    }
+    else
+    {
+      getInput<double>("behavior_time", behaviorTime_);
```

Comment:
@patapufate mind to explain the rationale behind this `behavior_time` parameter? I can't seem to find it in the `mission_manager` package.

Reply:
In the mission manager package the time_out parameter is used to finish the action. I have changed the name of the parameter to behavior_time to differenciate from time_out

Reply:
I see. Like a wait. Can't we implement such a wait in the tree? An force success action decorated with a delay would have the same effect, wouldn't it?